sylvia/tests/reply_generation.rs

```diff
@@ -0,0 +1,99 @@
+#![cfg(feature = "sv_replies")]
```

Comment:
Because it's breaking. Our current support for replies works for method signature with `_: Reply` as one of the parameters.
We will remove this feature in `2.0.0` and make it the default behavior.